client/packages/common/src/types/schema.ts

```diff
@@ -1470,11 +1464,8 @@ export type CustomerIndicatorInformationNodeCustomerArgs = {
 
 export type CustomerProgramRequisitionSettingNode = {
   __typename: 'CustomerProgramRequisitionSettingNode';
-  customerAndOrderTypes: Array<CustomerAndOrderTypeNode>;
-  masterList: MasterListNode;
-  programId: Scalars['String']['output'];
-  programName: Scalars['String']['output'];
-  tagName: Scalars['String']['output'];
+  customerNameId: Scalars['String']['output'];
+  masterLists: Array<MasterListWithOrderTypesNode>;
```

Comment:
Changing this node structure. I thought this would be more clear as we can have multiple program_requisition_settings per customer / program combination.

So we now query on a program_requisition_settings basis - and include the 'tag' in the selection as seen below. This is to differentiate between what might appear to be duplicate programs for each customer.